### PR TITLE
MGMT-1469 Add "Hosts not showing up" to the empty state

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

**Comment by mareklibra** [comment]
Cc @andybraren

**Review by andybraren — APPROVED** [review]
Looks great, thank you!

**Review by jtomasek — CHANGES_REQUESTED** [review]
This solution will cause the modal to close if a host gets registered in the meantime. Since we're opening the modal from 2 different places now, we should move the modal opening state from `DiscoveryTroubleshootingModalButton` to `BaremetalInventory` component level, render the DiscoveryTroubleshootingModal there and pass the 'showModal/setShowModal' as props to the button

File: src/components/hosts/HostsTable.tsx [hunk 1]
```diff
@@ -118,20 +123,35 @@ const hostToHostTableRow = (openRows: OpenRows, cluster: Cluster) => (host: Host
...
+  cluster: Cluster;
+  setDiscoveryHintModalOpen?: HostsNotShowingLinkProps['setDiscoveryHintModalOpen'];
+}> = ({ cluster, setDiscoveryHintModalOpen }) => (
   <EmptyState
     icon={ConnectedIcon}
     title="Waiting for hosts..."
     content="Hosts may take a few minutes to appear here after booting."
     primaryAction={<DiscoveryImageModalButton cluster={cluster} />}
+    secondaryActions={
+      setDiscoveryHintModalOpen && [
```
Comment (mareklibra) [inline src/components/hosts/HostsTable.tsx:136]: The use from `ClusterDetail` will never come to empty state ...

**Review by jtomasek — APPROVED** [review]

**Pull request merged**